### PR TITLE
Add SNMP examples to readme

### PR DESCRIPTION
**Pull request by sjwang90**
Used your sample config @kiler129 for the first example

### CODE REVIEW

**Comment by Hipska** [comment]
What is the goal of this? The README file already has more complete examples than the one added?

**Comment by sjwang90** [comment]
@Hipska There's been requests for more real world examples for the SNMP plugin #7433. I wanted to create a folder where people can add useful examples and link it from the readme. 

This single device example was one of the basic ones mentioned. If you think it's redundant, we can remove it.

**Comment by Hipska** [comment]
It's a good idea, but that example is just too simple, it only provides the system uptime. For real world examples I would at least expect one of cpu/memory/disk/temperature/interface stats.

**Comment by sjwang90** [comment]
Do you have any real-world examples you'd like to share? I'm totally good with getting rid of this PR since it wouldn't be that useful.

File: plugins/inputs/snmp/README.md [hunk 1]
```diff
@@ -231,6 +231,8 @@ interface,agent_host=127.0.0.1,ifDescr=wlan0,ifIndex=3,source=example.org ifAdmi
 interface,agent_host=127.0.0.1,ifDescr=eth0,ifIndex=2,source=example.org ifAdminStatus=1i,ifInDiscards=0i,ifInErrors=0i,ifInNUcastPkts=21i,ifInOctets=3852386380i,ifInUcastPkts=3634004i,ifInUnknownProtos=0i,ifLastChange=9088763i,ifMtu=1500i,ifOperStatus=1i,ifOutDiscards=0i,ifOutErrors=0i,ifOutNUcastPkts=0i,ifOutOctets=434865441i,ifOutQLen=0i,ifOutUcastPkts=2110394i,ifPhysAddress="c8:5b:76:c9:e6:8c",ifSpecific=".0.0",ifSpeed=1000000000i,ifType=6i 1575509815000000000
 interface,agent_host=127.0.0.1,ifDescr=lo,ifIndex=1,source=example.org ifAdminStatus=1i,ifInDiscards=0i,ifInErrors=0i,ifInNUcastPkts=0i,ifInOctets=51555569i,ifInUcastPkts=339097i,ifInUnknownProtos=0i,ifLastChange=0i,ifMtu=65536i,ifOperStatus=1i,ifOutDiscards=0i,ifOutErrors=0i,ifOutNUcastPkts=0i,ifOutOctets=51555569i,ifOutQLen=0i,ifOutUcastPkts=339097i,ifSpecific=".0.0",ifSpeed=10000000i,ifType=24i 1575509815000000000
 ```
+### Real-world Configuration Examples
+- [Single device](https://github.com/influxdata/telegraf/blob/snmp-examples/plugins/inputs/snmp/testdata/single-device.conf)
```
Comment (Hipska) [inline plugins/inputs/snmp/README.md:235]: I don't think that's a good link.
```suggestion
- [Single device](testdata/single-device.conf)
```

**Review by Hipska — CHANGES_REQUESTED** [review]

**Comment by Hipska** [comment]
I think I could forge some, but the problem actually is that each setup is very specific and most of the time, it isn't always purely snmp config, but also some processors to fix some data.

**Comment by sjwang90** [comment]
I think that's nice as well - if it's clear in the sample config to explain what the purpose of the processor is? Sample input and output data is always good.

**Comment by sjwang90** [comment]
@Hipska I'm going to close this - if you have any examples to add please do so! I'll assign you in the original issue. you are the SNMP master 👑

**Pull request closed without merging**